TShockAPI/Bouncer.cs

```diff
@@ -2496,5 +2511,45 @@ internal static int GetMaxPlaceStyle(int tileID)
 			{ ProjectileID.GoldShortswordStab, ItemID.GoldShortsword },
 			{ ProjectileID.PlatinumShortswordStab, ItemID.PlatinumShortsword }
 		};
+
+		private Dictionary<short, short> Projectile_MinValuesAI = new Dictionary<short, short> {
+			{ 611, -1 },
+
+			{ 950, 0 }
+		};
+		private Dictionary<short, short> Projectile_MaxValuesAI = new Dictionary<short, short> {
+			{ 611, 1 },
+
+			{ 950, 0 }
+		};
+
+		private Dictionary<short, short> Projectile_MinValuesAI2 = new Dictionary<short, short> {
+			{ 405, 0 },
+			{ 410, 0 },
+
+			{ 424, 0 },
+			{ 425, 0 },
+			{ 426, 0 },
+
+			{ 612, 0 },
+			{ 953, 0 },
+
+			{ 756, 0 },
+			{ 522, 0 }
+		};
+		private Dictionary<short, short> Projectile_MaxValuesAI2 = new Dictionary<short, short> {
+			{ 405, 2 },
+			{ 410, 2 },
+
+			{ 424, 1 },
+			{ 425, 1 },
+			{ 426, 1 },
+
+			{ 612, 2 },
```

Comment:
```suggestion
			{ 612, 0.7f },
```
Source: `Player.ItemCheck_Shoot()`, `float ai4 = (Main.rand.NextFloat() - 0.5f) * ((float)Math.PI / 4f);`
Increments every tick by `0.01`, has a lifetime of 60 for a total for +0.3. `Projectile.AI()`, `this.ai[1] += 0.01f;`